src/index.spec.ts

```diff
@@ -370,6 +372,34 @@ describe('ts-node', function () {
       cp.stdin!.end('\nconst a = 123\n.type a')
     })
 
+    it('REPL can be created via API', async () => {
+      const stdin = new PassThrough()
+      const stdout = new PassThrough()
+      const stderr = new PassThrough()
+      const replService = createReplService({
+        stdin,
+        stdout,
+        stderr
+      })
+      const service = create(replService.evalStateAwareHostFunctions)
```

Comment:
As a newcomer, I experienced some confusion seeing the `replService` juxtaposed with the `service`. I noticed that the `Register` type applies to the service and that naming discrepancy was also a head-scratcher for me. With some digging, I kind of see what they represent.

Nothing immediately applicable, but just sharing some thoughts, as I recall y'all considering what might make the repo easier on newbies!

Reply:
Yeah, I don't like the name of `Register`.  IMO it should be `Service` so that if you do `import {Service} from 'ts-node';`  We can rename it to `Service` and export by the legacy `Register` name for backwards-compatibility.

Reply:
Another question:
Given that our `create()` function returns a `Register`, which I'd like to rename to `Service`, does it make sense to rename `createReplService()` to `createRepl()` for consistency?  `create(): Service`, `createRepl(): ReplService`?

Reply:
Yes, I think that would make a lot more sense! I might even rename the `ReplService` type to `TsRepl`, or `TsNodeRepl`.

Reply:
I'm thinking it's best to avoid putting `TsNode` into the names of exports, to match TypeScript and node's APIs.  For example:

* `import {Duplex} from 'streams';` (it's called Duplex, not DuplexStream)
* `import {LanguageService} from 'typescript';` It's named LanguageService instead of TsLanguageService, and lots of usages I've seen do a wildcard import to add the `ts` prefix: `import * as ts from 'typescript'; ts.LanguageService`